templates/certified/index.html

```diff
@@ -0,0 +1,721 @@
+            alt="Ubuntu 认证硬件徽标",
+            width="90",
+            height="128",
+            hi_def=True,
+            loading="auto"
+          ) | safe }}
+        </div>
+        <div class="col">
+          <h1>Ubuntu 硬件认证计划</h1>
+          <h2>
+            通过广泛的测试和审查过程，<br/>
+            为您确保最佳的Ubuntu使用体验
+          </h2>
+          <p>
+            通过 Ubuntu 官方认证的硬件，如服务器、笔记本和工作站、IoT物联网设备等等可以获得更好的支持，开箱即用的体验，
+            且无需为繁琐的系统组件驱动和兼容性而担忧。
+          </p>
+          <hr class="p-rule--muted" />
+          <a class="p-button--positive" href="/contact">与我们联系</a>
+        </div>
+      </div>
+  </section>
+
+  {%- call(slot) vf_tiered_list(
+    is_description_full_width_on_desktop=false,
+    is_list_full_width_on_tablet=true,
+    is_image_full_width=false
+  ) -%}
+    {%- if slot == 'title' -%}
+      <h2>通过认证的硬件享最佳<br/>Ubuntu体验</h2>
+    {%- endif -%}
+
+    {%- if slot == 'description' -%}
+      <p>预装开箱即用的Ubuntu操作系统，并已通过广泛的可靠性和性能测试</p>
+    {%- endif -%}
+
+    {%- if slot == "image" -%}
+      <div class="p-image-container--16-9 is-cover">
+        {{ image(
+          url="https://assets.ubuntu.com/v1/966f0d9f-laptop.png",
+          alt="",
+          width="1200",
+          height="675",
+          hi_def=True,
+          loading="auto",
+          attrs={"class": "p-image-container__image"}
+        ) | safe }}
+      </div>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_1' -%}
+      <h3 class="p-heading--5">选择值得信赖的硬件</h3>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_1' -%}
+      <p>查看Ubuntu认证机型列表，选择符合您需求的合作伙伴硬件。</p>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_2' -%}
+      <h3 class="p-heading--5">降低成本</h3>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_2' -%}
+      <p>可靠的认证硬件可簡化解决方案的实施，降低操作系统开发和支持的成本。</p>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_3' -%}
+      <h3 class="p-heading--5">加速产品上市</h3>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_3' -%}
+      <p>利用 Canonical 和硬件合作伙伴的专业知识，快速将您的解决方案推向市场。</p>
+    {%- endif -%}
+  {%- endcall -%}
+
+  <section class="p-strip">
+    <div class="p-strip u-no-padding--top">
+      <div class="grid-row">
+        <hr class="p-rule" />
+        <div class="grid-col-4">
+          <h2 id="certified-testing-heading">广泛的测试和验证</h2>
+        </div>
+        <div class="grid-col-4">
+          <h5>确保驱动程序集成和出色的设备性能</h5>
+          <p>
+            Canonical 的认证团队进行超过 500 项专注于操作系统兼容性的硬件测试，以确保 Ubuntu 始终为您提供最佳体验。系统的各个方面，
+            包括驱动程序集成等，都经过全面验证把关。
+          </p>
+        </div>
+      </div>
+    </div>
+
+    <ul class="grid-row" aria-labelledby="certified-testing-heading">
+      <li class="grid-col-2 grid-col-medium-2 grid-col-small-2">
+        <p class="p-certified-testing-item">
+          {{ image(url="https://assets.ubuntu.com/v1/913fdc9e-audio.svg",
+            alt="",
+            width="40",
+            height="40",
+            hi_def=True,
+            loading="auto"
```

Comment:
As these images are below the fold 
```suggestion
            loading="lazy"
```
Any images below the fold can be lazy loaded. So this and all the ones below

Reply:
I needed to upgrade the image template to a version that [includes this change](https://github.com/canonical/canonicalwebteam.image-template/pull/61) so that an intermediate block-displayed `div.lazyloaded` isn't generated, which would throw off the inline layout of these items. It's working great now with lazyloading.